client/lib/query-manager/theme/key.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import omitBy from 'lodash/omitBy';
+
+/**
+ * Internal dependencies
+ */
+import PaginatedQueryKey from '../paginated/key';
+import { DEFAULT_THEME_QUERY } from './constants';
+
+/**
+ * Returns true if the specified key value query pair is identical to that of
+ * the default theme query, is null, or is undefined.
+ *
+ * @param  {*}       value Value to check
+ * @param  {String}  key   Key to check
+ * @return {Boolean}       Whether key value matches default query or is null
+ */
+function isDefaultOrNullQueryValue( value, key ) {
+	return (
+		null == value || // Double-equals null checks undefined, null
```

Comment:
Mhm, good point.

>  Also, would a general truthiness check here suffice? I think with the other implementations I had worried that empty string or 0 could have an impact on query results, but that's not necessarily always the case.

`filter` can be `''`, but that's its default value anyway, so that wouldn't really make a difference. `search` could be both `''` (again default) and `0`, which isn't its default value.

Leaning towards an explicit 

```js
value === undefined || value === null
````
check.